### ISSUE TITLE
init script reports errors on Ubuntu 16.04

### ISSUE THREAD

**Issue by matthew-hollick** [go-graphite/go-carbon#189]
Hi,

The provided init script does not work well with the version of ubuntu that we run in house. I suspect that the behaviour will be the same with all recent ubuntu versions.

```
carbon-1: /etc/init.d/ # systemctl enable go-carbon
Synchronizing state of go-carbon.service with SysV init with /lib/systemd/systemd-sysv-install...
Executing /lib/systemd/systemd-sysv-install enable go-carbon
insserv: warning: script 'K01go-carbon' missing LSB tags and overrides
insserv: warning: script 'go-carbon' missing LSB tags and overrides
update-rc.d: error: go-carbon Default-Start contains no runlevels, aborting.
```

Although an error is reported the service appears to have been correctly registered as enabled in systemd.
Amending the init script to include the following block at the top removes the error:

```
carbon-1: /etc/init.d/ # head go-carbon
#!/bin/sh
### BEGIN INIT INFO
# Provides:          go-carbon
# Required-Start:    $all
# Required-Stop:     0 1 6
# Default-Start:     2 3 4 5
# Default-Stop:      0 1 6
# Short-Description: go-carbon carbon daemon
### END INIT INFO

carbon-1: /etc/init.d/ # systemctl enable go-carbon
Synchronizing state of go-carbon.service with SysV init with /lib/systemd/systemd-sysv-install...
Executing /lib/systemd/systemd-sysv-install enable go-carbon
```

As I don't know how the lsb preamble in the current init file is used I have refrained from creating a PR for this error.

Cheers!

**Comment by towolf** [comment]
I've had this issue as well. I've resorted to dpkg-diverting the file in /etc/init.d to another location.

**Comment by lomik** [comment]
Released in 0.11.0
